Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/runtime/conversion_generator.go

```diff
@@ -185,11 +185,15 @@ func isComplexType(reflection reflect.Type) bool {
 func (g *conversionGenerator) generateConversionsForMap(inType, outType reflect.Type) error {
 	inKey := inType.Key()
 	outKey := outType.Key()
+	g.addImportByPath(inKey.PkgPath())
```

Comment:
I'm not very familiar with the generators, but I would have expected a corresponding deep copy change.  Why isn't it needed?